subsys/bluetooth/host/conn.c

```diff
@@ -4163,3 +4174,23 @@ void bt_hci_le_df_cte_req_failed(struct net_buf *buf)
 #endif /* CONFIG_BT_DF_CONNECTION_CTE_REQ */
 
 #endif /* CONFIG_BT_CONN */
+
+#if defined(CONFIG_BT_CONN_TX_WQ)
+static int bt_conn_tx_workq_init(void)
+{
+	static const struct k_work_queue_config cfg = {
```

Comment:
You are right. I made it static to avoid allocating the structure on stack and thanks to that save a few bytes of FLASH.
Anyway, changed as suggested.